src/calculateRentalCost.js

```diff
@@ -5,6 +5,28 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const PRICE_PER_DAY = 40;
+  const VERY_SHORT_TERM = 2;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCAUNT = 50;
+  const OVER_SHORT_TERM = 6;
+  const PRE_SHORT_TERM = 3;
+  const SHORT_TERM_DISCAUNT = 20;
```

Comment:
The constants 'VERY_SHORT_TERM', 'OVER_SHORT_TERM', and 'PRE_SHORT_TERM' are not necessary and are not used correctly in the logic. It's better to use only 'SHORT_TERM' for 3 days threshold and 'LONG_TERM' for 7 days threshold, removing the extra constants to avoid confusion.